pkg/virt-controller/watch/pool/pool.go

```diff
@@ -568,58 +570,51 @@ func sortVMsRandom(vms []*virtv1.VirtualMachine) {
 
 func (c *Controller) scaleIn(pool *poolv1.VirtualMachinePool, vms []*virtv1.VirtualMachine, count int) error {
 
-	poolKey, err := controller.KeyFunc(pool)
-	if err != nil {
-		return err
-	}
-
-	elgibleVMs := filterRunningVMs(vms)
+	eligibleVMs := filterRunningVMs(vms)
 
 	// make sure we count already deleting VMs here during scale in.
-	count = count - (len(vms) - len(elgibleVMs))
+	count = count - (len(vms) - len(eligibleVMs))
+
+	return c.proactiveScaleIn(pool, eligibleVMs, count)
+}
 
-	if len(elgibleVMs) == 0 || count == 0 {
+func (c *Controller) proactiveScaleIn(pool *poolv1.VirtualMachinePool, vms []*virtv1.VirtualMachine, count int) error {
```

Comment:
Shouldn't this be called `scaleIn` and the method in L571 `proactiveScaleIn` instead?